logic/account.js

```diff
@@ -436,6 +314,188 @@ function Account (db, schema, logger, cb) {
 	return setImmediate(cb, null, this);
 }
 
+Account.prototype.schema = {
+	id: 'Account',
+	type: 'object',
+	properties: {
+		username: {
+			type: 'string',
+			case: 'lower',
+			maxLength: 20,
+			minLength: 1
+		},
+		isDelegate: {
+			type: 'integer',
+			maximum: 32767
+		},
+		u_isDelegate: {
+			type: 'integer',
+			maximum: 32767
+		},
+		secondSignature: {
+			type: 'integer',
+			maximum: 32767
+		},
+		u_secondSignature: {
+			type: 'integer',
+			maximum: 32767
+		},
+		u_username: {
+			anyOf: [
+				{
+					type: 'string',
+					case: 'lower',
+					maxLength: 20,
+					minLength: 1
+				},
+				{
+					type: 'null'
+				}
+			]
+		},
+		address: {
+			type: 'string',
+			case: 'upper',
+			minLength: 1,
+			maxLength: 22
+		},
+		publicKey: {
+			type: 'string',
+			format: 'publicKey'
+		},
+		secondPublicKey: {
+			anyOf: [
+				{
+					type: 'string',
+					format: 'publicKey'
+				},
+				{
+					type: 'null'
+				}
+			]
+		},
+		balance: {
+			type: 'integer',
+			minimum: 0,
+			maximum: constants.totalAmount
+		},
+		u_balance: {
+			type: 'integer',
+			minimum: 0,
+			maximum: constants.totalAmount
+		},
+		rate: {
+			type: 'integer'
+		},
+		delegates: {
+			anyOf: [
+				{
+					type: 'array',
+					uniqueItems: true
+				},
+				{
+					type: 'null'
+				}
+			]
+		},
+		u_delegates: {
+			anyOf: [
+				{
+					type: 'array',
+					uniqueItems: true
+				},
+				{
+					type: 'null'
+				}
+			]
+		},
+		multisignatures: {
```

Comment:
Change related multisig fields to be same as `Multisignature.prototype.schema`
- multisignatures, u_multisignatures is `keygroup`
- multimin, u_ multimin is `min`
- multilifetime, u_ multilifetime is `lifetime`